xml/System.Text.Json/JsonSerializerOptions.xml

```diff
@@ -63,8 +63,8 @@ For more information, see [How to serialize and deserialize JSON](/dotnet/standa
         <Parameter Name="defaults" Type="System.Text.Json.JsonSerializerDefaults" Index="0" FrameworkAlternate="net-5.0" />
       </Parameters>
       <Docs>
-        <param name="defaults">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="defaults">The <see cref="T:System.Text.Json.JsonSerializerDefaults" /> to reason about.</param>
+        <summary>Constructs a new <see cref="T:System.Text.Json.JsonSerializerOptions" /> instance with a predefined set of options determined by the specified <see cref="T:System.Text.Json.JsonSerializerDefaults" />.</summary>
         <remarks>To be added.</remarks>
```

Comment:
Can we please add this to the remarks section?

> By default, serialization does not support objects with cycles and does not preserve duplicate references. Metadata properties will not be written when serializing reference types and will be treated as regular properties on deserialize.
> 
> * On Serialize:
>   * Treats duplicate object references as if they were unique and writes all their properties.
>   * The serializer throws a <see cref="JsonException"/> if an object contains a cycle.
> * On Deserialize:
>   * Metadata properties (`$id`, `$values`, and `$ref`) will not be consumed and therefore will be treated as regular JSON properties.
>   * The metadata properties can map to a real property on the returned object if the property names match, or will be added to the <see cref="JsonExtensionDataAttribute"/> overflow dictionary, if one exists; otherwise, they are ignored.